tests/Plugin_Context_Tests.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Tests for the Plugin_Context class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Plugin_Context;
+
+class Plugin_Context_Tests extends WP_UnitTestCase {
+	public function set_up() {
+		parent::set_up();
+
+		$this->plugin_context = new Plugin_Context( 'plugin-check/plugin-check.php' );
+	}
+
+	public function test_basename() {
+		$basename = $this->plugin_context->basename();
+
+		$this->assertSame( $basename, 'plugin-check/plugin-check.php' );
```

Comment:
```suggestion
		$this->assertSame( 'plugin-check/plugin-check.php', $this->plugin_context->basename() );
```

As per the [assertSame documentation](https://phpunit.readthedocs.io/en/9.5/assertions.html#assertsame) the expected should be in first place. Also remove variable.

Do similar change for other tests.